src/distilabel/llm/ollama.py

```diff
@@ -116,22 +163,23 @@ def model_name(self) -> str:
         return self.model
 
     def _api_available(self):
-        """Calls GET {OLLAMA_HOST}"""
-        msg = f"Could not connect to Ollama as {self.OLLAMA_HOST}. Check https://github.com/ollama/ollama for deployment guide."
+        """Checks if the Ollama API is available."""
+        msg = f"Could not connect to Ollama at {self.OLLAMA_HOST}. Check https://github.com/ollama/ollama-python/tree/main for deployment guide."
         try:
-            response = request.urlopen(self.OLLAMA_HOST)
-            if response.getcode() != 200:
-                raise Exception
-        except Exception as e:
+            ollama.list()
+        except ollama.ResponseError as e:
             raise ValueError(msg) from e
```

Comment:
Here I would remove the `msg` variable as only used once, and directly add it within `raise ValueError(...)`